src/main/java/com/carservice/manager/service/VehicleDbService.java

```diff
@@ -0,0 +1,70 @@
+package com.carservice.manager.service;
+
+import com.carservice.manager.config.Log4J2YamlConfig;
+import com.carservice.manager.entity.VehicleEntity;
+import com.carservice.manager.model.VehicleModel;
+import com.carservice.manager.repository.VehicleDbRepository;
+import com.carservice.manager.utils.VehicleWrapper;
+import org.springframework.stereotype.Service;
+import org.springframework.validation.annotation.Validated;
+
+import javax.transaction.Transactional;
+import java.util.Date;
+import java.util.List;
+import java.util.UUID;
+
+@Validated
+@Service
+public class VehicleDbService {
```

Comment:
It would be good to introduce an interface for VehicleSerices, so we can define all needed methods there. And then declare this interface in controllers 😉 
```suggestion
public class VehicleDbService implements VehicleService {
```